src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/service/PublicBookerService.kt

```diff
@@ -25,11 +25,26 @@ class PublicBookerService(
   private val prisonerSearchService: PrisonerSearchService,
   private val prisonerContactService: PrisonerContactService,
   private val prisonService: PrisonService,
+  private val prisonerInfoClient: PrisonerInfoClient,
 ) {
   companion object {
     val logger: Logger = LoggerFactory.getLogger(this::class.java)
     const val PRISONER_VALIDATION_ERROR_MSG = "prisoner validation for prisoner number - {0} failed with error - {1}"
     const val PRISON_VALIDATION_ERROR_MSG = "prison validation for prison code - {0} for prisoner number - {1} failed with error - {2}"
+
+    fun validatePrison(prison: VisitSchedulerPrisonDto, prisonService: PrisonService): String? {
```

Comment:
Could this go into a PublicPrisonUtil class ? As it's shared between this and the prisonerInfoClient ?

Reply:
Updated